api/routes/v1/companies.js

```diff
@@ -0,0 +1,33 @@
+const router = require('express').Router()
+const Companies = require('../../models/v1/companies')
+
+// [GET /api/v1/companies]
+// [GET /api/v1/companies?name=[partial-query]]
+// [GET /api/v1/companies?employees_lte=[integer]]
+// [GET /api/v1/companies?employees_gte=[integer]]
+router.get('/', async (req, res, next) => {
+  try {
+    const status = 200
+    let query =  req.query
+    const {employees_lte, employees_gte} = req.query
+    if (employees_lte) {
+      query['$lte']=employees_gte 
+      delete query.employees_lte
```

Comment:
You also wouldn't want to delete the other query parameter in case you had both.